examples/bri-3/src/bri/auth/strategy/jwt.strategy.ts

```diff
@@ -0,0 +1,20 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
```

Comment:
importing nestjs/passport
passport
passport-jwt
nestjs/jwt

in same project? do we need all those packages?

Reply:
I would suggest reading through this document: https://docs.nestjs.com/security/authentication#authentication